.gitlab-ci.yml

```diff
@@ -21,6 +21,7 @@ stages:
   - pre_release_test
   - release
   - release_image
+  - post_release_image
```

Comment:
Right now, our release stage:
- pushes shared and data-pipelines to npm registry
- pushes the ui docker image to docker registry

We want to:
- re-deploy the helm chart for the test environment **after** a new docker image is released
- ping the frontend in the newly-deployed helm chart **after** deployment finished

It seemed cleaner this way and we're not messing with existing stages.

Reply:
Do we utilise release candidates, to deploy and test, before making an official release?

Reply:
Yes and no. Whatever is tagged as latest will go to the test environment. We can run tests on it and then tag as stable.

Reply:
We are using semantic versioning https://semver.org, that provides explicit support for release candidates, in format 1.0.0-rc.1 https://github.com/vmware/versatile-data-kit/blob/main/CONTRIBUTING.md#semantic-versioning

Reply:
No need for release candidates, imo. We've discussed before that the docker image is not considered a release artefact. The helm chart is the artefact we release. Having a docker image that is broken is fine, as long as it doesn't break the helm chart. In this case, it won't, because the helm chart will always use the stable tag.